client/src/test/java/GeneralSMATest.java

```diff
@@ -0,0 +1,34 @@
+import com.fiskaly.kassensichv.sma.GeneralSMA;
+
+import java.io.IOException;
+
+public class GeneralSMATest {
+
+    public static GeneralSMA generalSMA = null;
+
+    public static void main(String[] args) {
```

Comment:
Suggestion: add a `throws Exception` to the `main` method's signature.
Then you don't have to `try/catch` within the method's body.
Instead, any runtime exception that might be thrown will crash the process, which is exactly what we want in this particular case (i.e. a test)... :)

Reply:
This makes a lot of sense, thanks for pointing that out. Changed bd854cb27f34dd37414900677d5b6cb7960dba01